Can't build the plugin

on hyprland 0.45.0
```
 last 25 log lines:
       >     includedir   : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/include
       >     infodir      : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/share/info
       >     libdir       : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/lib
       >     libexecdir   : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/libexec
       >     localedir    : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/share/locale
       >     mandir       : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/share/man
       >     prefix       : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1
       >     sbindir      : /nix/store/fq3kdcdy2v9hmfxv2q3yzkv4whnq7g91-split-monitor-workspaces-0.1/sbin
       >     wrap_mode    : nodownload
       >
       > Found ninja-1.12.1 at /nix/store/nwsn9hg84pf9rpy3bgdpssa2b6qc7w3p-ninja-1.12.1/bin/ninja
       > mesonConfigurePhase: enabled\ parallel\ building
       > Running phase: buildPhase
       > build flags: -j16
       > [1/2] Compiling C++ object libsplit-monitor-workspaces.so.p/src_main.cpp.o
       > FAILED: libsplit-monitor-workspaces.so.p/src_main.cpp.o
       > g++ -Ilibsplit-monitor-workspaces.so.p -I. -I.. -I../include -I/nix/store/w0lm91q71vypip8jyqc670hsk2w0nsds-hyprland-0.45.0+date=2024-12-03_320144a-dev/include -I/nix/store/w0lm91q71vypip8jyqc670hsk2w0nsds-hyprland-0.45.0+date=2024-12-03_320144a-dev/include/hyprland/protocols -I/nix/store/w0lm91q71vypip8jyqc670hsk2w0nsds-hyprland-0.45.0+date=2024-12-03_320144a-dev/include/hyprland -I/nix/store/f40qqcxl63lgfygc22x1h9n9vyd76zcz-pixman-0.43.4/include/pixman-1 -I/nix/store/kypqq0shrqvmjpk293cmk4jzw9g20nwq-libdrm-2.4.123-dev/include -I/nix/store/kypqq0shrqvmjpk293cmk4jzw9g20nwq-libdrm-2.4.123-dev/include/libdrm -fdiagnostics-color=always -D_GLIBCXX_ASSERTIONS=1 -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -std=c++23 -DWLR_USE_UNSTABLE -fno-gnu-unique -fPIC -MD -MQ libsplit-monitor-workspaces.so.p/src_main.cpp.o -MF libsplit-monitor-workspaces.so.p/src_main.cpp.o.d -o libsplit-monitor-workspaces.so.p/src_main.cpp.o -c ../src/main.cpp
       > ../src/main.cpp:21:7: error: 'CColor' does not name a type
       >    21 | const CColor s_pluginColor = {0x61 / 255.0F, 0xAF / 255.0F, 0xEF / 255.0F, 1.0F};
       >       |       ^~~~~~
       > ../src/main.cpp: In function 'void raiseNotification(const std::string&, float)':
       > ../src/main.cpp:39:56: error: 's_pluginColor' was not declared in this scope
       >    39 |         HyprlandAPI::addNotification(PHANDLE, message, s_pluginColor, timeout);
       >       |                                                        ^~~~~~~~~~~~~
       > ninja: build stopped: subcommand failed.
```
Im assuming this has to do with [this change](https://github.com/hyprwm/Hyprland/commit/320144ae7288fe23686935ebb235d9fe0c900862)

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -18,7 +18,7 @@ auto constexpr k_keepFocused = "plugin:split-monitor-workspaces:keep_focused";
 auto constexpr k_enableNotifications = "plugin:split-monitor-workspaces:enable_notifications";
 auto constexpr k_enablePersistentWorkspaces = "plugin:split-monitor-workspaces:enable_persistent_workspaces";
 
-const CColor s_pluginColor = {0x61 / 255.0F, 0xAF / 255.0F, 0xEF / 255.0F, 1.0F};
+const CHyprColor s_pluginColor = {0x61 / 255.0F, 0xAF / 255.0F, 0xEF / 255.0F, 1.0F};
 bool g_enableNotifications = false;
 bool g_enablePersistentWorkspaces = true;
 bool g_keepFocused = false;

